README.md

```diff
@@ -19,6 +19,15 @@ npm i
 npm run download-docs -w nextjs-website
 ```
 
+#### How to speed up local docs fetch time
+
+Initialize a git submoudule on docs directory:  
+`rm -R apps/nextjs-website/docs && git submodule add --force https://github.com/pagopa/devportal-docs.git apps/nextjs-website/docs`  
+
+Run the script to checkout docs to the right branch
+`npm run update-docs -w nextjs-website` 
```

Comment:
You didn't add the two options. If a developer runs the commands in the order written in the README file, they will download a gigabyte of data and then delete it